src/sql/workbench/services/profiler/browser/profilerService.ts

```diff
@@ -319,12 +320,15 @@ export class ProfilerService implements IProfilerService {
 			const fileURI = fileURIs[0];
 
 			try {
-				let fileSize = await (await fs.stat(fileURI.fsPath)).size;
-				console.log(fileSize);
-				const fileLimitSize = 1073741824; 	// 1GB
+				const fileSize = (await fs.stat(fileURI.fsPath)).size;
+				const fileLimitSize = 1 * ByteSize.GB;
+				const fileOpenWarningSize = 100 * ByteSize.MB;
+
 				if (fileSize > fileLimitSize) {
-					this._notificationService.error(nls.localize('FileTooLarge', "File is too large to open and it may hang ADS. "));
+					this._notificationService.error(nls.localize('FileTooLarge', "The file is too large to open in profiler. The profiler can open files that are less than 1GB."));
 					return false;
+				} else if (fileSize > fileOpenWarningSize) {
+					this._notificationService.info(nls.localize('LargeFileWait', "Loading the file might take a moment, because the file is large."));
```

Comment:
nit: update wording to `Loading the file might take a moment due to the file size.`